src/components/index.js

```diff
@@ -333,12 +339,13 @@ export default class Rnd extends Component {
         onDrag={this.onDrag}
         onStop={this.onDragStop}
         axis={this.props.dragAxis}
+        disabled={this.props.disableDragging}
```

Comment:
Please add type annotation to `type Props` like below.

```
type Props = {
    ...
    disableDragging: boolean;
    ...
}
```

And pleaase fix some flowtype error. you can check flowtype running  `npm run flow` 🙏